Aztec/Classes/GUI/FormatBar/FormatBar.swift

```diff
@@ -576,6 +630,16 @@ private extension FormatBar {
     /// Sets up the Constraints
     ///
     func configureConstraints() {
+        var leadingAnchor = self.leadingAnchor
+        var trailingAnchor = self.trailingAnchor
+        var bottomAnchor = self.bottomAnchor
+
+        if #available(iOS 11.0, *) {
+            leadingAnchor = safeAreaLayoutGuide.leadingAnchor
```

Comment:
ins't the safeAreaLayoutGuide the same as layoutMarginsGuide on iOS11? any difference on the value?

Reply:
As far as I'm aware, the layout margins guide is the same as the safe area + margins.

Reply:
Ok so the layoutMargins has extra spacing.